lib/src/enforcer.dart

```diff
@@ -78,5 +78,48 @@ class Enforcer extends CoreEnforcer {
     return _enforcer;
   }
 
-  Model newModel(List<String> text) => Model();
+  /// getRolesForUser gets the roles that a user has.
+  ///
+  /// [param] name the user.
+  /// [return] the roles that the user has.
+  List<String> getRolesForUser(String name) {
+    try {
+      return model.model['g']!['g']!.rm.getRoles(name);
+    } catch (e) {
+      if (e.toString() == 'Null check operator used on a null value') {
+        throw Exception("model does not contain 'g' section");
```

Comment:
You can use `FormatException` here, I would discourage you from raising the `Exception` class directly, always throw a more specific `Exception` or `Error`.